docker-compose.yml

```diff
@@ -38,13 +57,13 @@ services:
       #- PUID=1000
       #- PGID=1000
       # PHP timezone e.g. PHP_TZ=America/New_York
-      - PHP_TZ=UTC
-      - TIMEZONE=UTC
+      - PHP_TZ=${TIMEZONE:-UTC}
+      - TIMEZONE=${TIMEZONE:-UTC}
       #- APP_NAME=Laravel
       #- APP_ENV=local
       #- APP_DEBUG=true
       #- APP_FORCE_HTTPS=false
-      #- APP_URL=http://localhost
+      - APP_URL=${APP_URL:-http://localhost}
```

Comment:
This is already the default - does this one not pass through?

Reply:
it makes it easier to modify in `.env` without having to fiddle in `lychee/config/.env` as misconfiguration of this value is the first cause of failure.

Reply:
But it's an env value, so wouldn't it work when modified in `.env` anyway?

Reply:
what do you mean ?

Reply:
I thought the idea was that setting `APP_URL` in .env set the environment variable in the container rather than in the shell? Which would mean this **should** pass through without adding that line.